3.3/examples/django/application-template-stibuild.json

```diff
@@ -0,0 +1,282 @@
+            ],
+            "from": {
+              "name": "django-example"
+            },
+            "tag": "latest"
+          },
+          "type": "ImageChange"
+        }
+      ]
+    },
+    {
+      "apiVersion": "v1beta1",
+      "containerPort": 5432,
+      "id": "database",
+      "kind": "Service",
+      "port": 5434,
+      "selector": {
+        "name": "database"
+      }
+    },
+    {
+      "apiVersion": "v1beta1",
+      "kind": "DeploymentConfig",
+      "metadata": {
+        "name": "database"
+      },
+      "template": {
+        "controllerTemplate": {
+          "podTemplate": {
+            "desiredState": {
+              "manifest": {
+                "containers": [
+                  {
+                    "env": [
+                      {
+                        "name": "POSTGRESQL_USER",
+                        "value": "${POSTGRESQL_USER}"
+                      },
+                      {
+                        "name": "POSTGRESQL_PASSWORD",
+                        "value": "${POSTGRESQL_PASSWORD}"
+                      },
+                      {
+                        "name": "POSTGRESQL_DATABASE",
+                        "value": "${POSTGRESQL_DATABASE}"
+                      }
+                    ],
+                    "image": "openshift/postgresql-92-centos7",
+                    "name": "django-demo-database",
+                    "ports": [
+                      {
+                        "containerPort": 5432
+                      }
+                    ]
+                  }
+                ],
+                "version": "v1beta1"
+              }
+            },
+            "labels": {
+              "name": "database"
+            }
+          },
+          "replicaSelector": {
+            "name": "database"
+          },
+          "replicas": 1
+        },
+        "strategy": {
+          "type": "Recreate"
+        }
+      },
+      "triggers": [
+        {
+          "type": "ConfigChange"
+        }
+      ]
+    }
+  ],
+  "kind": "Template",
+  "metadata": {
+    "annotations": {
+      "description": "This example shows how to create a simple Django application in OpenShift Origin v3",
+      "tags": "python,django,postgres",
+      "iconClass" : "icon-python"
+    },
+    "name": "django-demo-application"
+  },
+  "parameters": [
+    {
+      "description": "Django WSGI application",
+      "value": "demo.wsgi",
+      "name": "APP_MODULE"
+    },
+    {
+      "description": "database username",
+      "from": "user[A-Z0-9]{3}",
+      "generate": "expression",
+      "name": "POSTGRESQL_USER"
+    },
+    {
+      "description": "database password",
+      "from": "[a-zA-Z0-9]{8}",
+      "generate": "expression",
+      "name": "POSTGRESQL_PASSWORD"
+    },
+    {
+      "description": "database name",
+      "name": "POSTGRESQL_DATABASE",
+      "value": "django_demo"
+    }
+  ],
+  "labels": {
+    "template": "application-template-stibuild"
+  }
+}
```

Comment:
If this is example, maybe before each object put a detailed comment what it does, and its purpose. Then in the README you'd point to this file and its comments. Other option is to copy parts of this file into the README, but the latter is more cumbersome in the long run when it comes to updating, and having this in one place is simpler.